states/AR/governor.yaml

```diff
@@ -0,0 +1,64 @@
+contact_form:
+  steps:
+    - visit: "https://arkansasgov.wufoo.com/embed/z10gjhti00fynaz/def/embedKey=z10gjhti00fynaz606148&entsource=&referrer=https%3Awuslashwuslashgithub.comwuslashresistbotwuslashcontact-officialswuslashissueswuslash71&header=hide"
```

Comment:
I think we can use a shorter URL here.

First thought is, when we request 
`https://arkansasgov.wufoo.com/embed/z10gjhti00fynaz/def/embedKey=z10gjhti00fynaz333575&entsource=&referrer=https%3Awuslashwuslashwww.google.comwuslash&header=hide`
it's a redirect to 
`https://arkansasgov.wufoo.com/forms/?formname=z10gjhti00fynaz&embed=1&embedKey=z10gjhti00fynaz333575&entsource=&referrer=https:wuslashwuslashwww.google.comwuslash&header=hide`

Then we can remove all the parameters except formname and still get the same form, so I think we can use this URL:

https://arkansasgov.wufoo.com/forms/?formname=z10gjhti00fynaz